src/Tools/dotnet-monitor/LoggingExtensions.cs

```diff
@@ -131,6 +131,18 @@ internal static class LoggingExtensions
                 logLevel: LogLevel.Critical,
                 formatString: "{failure}");
 
+        private static readonly Action<ILogger, Exception> _excessivePermissions =
+            LoggerMessage.Define(
+                eventId: new EventId(21, "ExcessivePermissions"),
+                logLevel: LogLevel.Warning,
+                formatString: "The process was launched at an administrative level and will have access to all processes on the system. The dotnet-monitor process should be launched as the same user as the target dotnet process.");
```

Comment:
I'm not completely sure this is how we want to phrase this guidance. For example, if the user wants to examine the state of w3wp processes, i believe they are forced to run as admin but they just can't use ntlm auth for that scenario.

Reply:
Should this just be `The process was launched as an administrator level and will have access to all processes on the system.`

Reply:
@shirhatti, do you have any opinion on the wording of this warning?

Reply:
Actually phrasing aside, the desired outcome is that a user doesn't run in any elevated context on accident. Simply put, only elevate if you need to.

As for my attempt, I'd probably say 

> The process was launched as an administrator and will have access to all processes on the system. Do not run elevated unless you need to monitor processes launched by other user (e.g., IIS worker processes)